arrays/array_practice.rb

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
+
+myarray = [1,2,3,4,5,6,7,8,9,10]
+myarray.each do |number|
```

Comment:
If only there was a method to _join_ elements of an array together with a string... :)

    myarray._______("...") + "..."